nibabel/tests/test_parrec.py

```diff
@@ -777,3 +779,11 @@ def test_exts2par():
                    list(nii_img.header.extensions)):
         hdrs = exts2pars(source)
         assert_equal(len(hdrs), 2)
+
+
+def test_dualTR():
+    with open(DUAL_TR_PAR, 'rt') as fobj:
+        with suppress_warnings():
```

Comment:
Sorry to ask, but would you mind also checking the warning is raised?   Something like:

```
    with clear_and_catch_warnings(modules=[parrec], record=True) as wlist:
        warnings.simplefilter('always')
        dualTR_hdr = PARRECHeader.from_fileobj(fobj)
        assert_equal(len(wlist), 1)
```